src/less-style/main.less

```diff
@@ -12,3 +12,7 @@
 @import "question-props";
 @import "structure";
 @import "tree-content";
+
+a {
```

Comment:
I wasn't sure where to put this rule, since there doesn't seem to be any place that contains global styles.

Reply:
Generic styles are in [/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/](https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/); I'd put this in typography.less.

Reply:
https://github.com/dimagi/commcare-hq/pull/24141